packages/compass-components/src/components/leafy-green/README.md

```diff
@@ -2,15 +2,10 @@
 
 > Copied from the [Leafy Green][leafy-green-repository] repository components.
 
-The most recent versions of Leafy Green comments cause all sorts of breaking changes and bring new branding designs that we currently postpone with enabling in Compass. The LG team hasn’t backported fixes in the past to the older versions of components, therefore to solve critical issues we decided to temporarily copy the broken LG components to our project to fix bugs on our end as a workaround.
+This folder hosts temporary patched copies of LG components that require workaround to work properly in Compass.
```

Comment:
changed this text since now we are at the latest version, and also made it more generic so there are less chances of this going stale

Reply:
why do we still use the copied Combobox? can't we switch to a new one already?